pysrc/bytewax/connectors/files.py

```diff
@@ -101,6 +104,50 @@ def build_part(self, for_part, resume_state):
         assert for_part == str(self._path), "Can't resume reading from different file"
         return _FileSource(self._path, resume_state)
 
+class CSVInput(FileInput):
```

Comment:
I am not sure it makes sense to provide a non-header version right now because that would be the same as lines that you could parse downstream with `next(csv.reader([line]))` if you don't care about the header. 

I'd like to leave it as is for right now and we can always add a way to skip the header or provide your own header.